doc/api/next_api_changes/behavior/18193-BKB.rst

```diff
@@ -0,0 +1,11 @@
+ID attribute of XML tags in SVG files now based on SHA512 rather than MD5
```

Comment:
But it's using `sha256`.

Reply:
Thanks! I had gone back and forth about which SHA to go for. I'm glad you caught this mistake!